### PR TITLE
Log switching to LDAP backup host because main one is not available

### PR DESCRIPTION
**Pull request by come-nc**
Signed-off-by: Côme Chilliet <come.chilliet@nextcloud.com>

### CODE REVIEW

**Comment by come-nc** [comment]
@blizzz Should that be warning? It’s an important information that main LDAP server is not reachable.

**Comment by come-nc** [comment]
Drone failure is not related.

File: apps/user_ldap/lib/Connection.php [hunk 1]
```diff
@@ -408,7 +408,7 @@ private function doSoftValidation() {
 				$uuidAttributes = Access::UUID_ATTRIBUTES;
 				array_unshift($uuidAttributes, 'auto');
 				if (!in_array($this->configuration->$effectiveSetting,
-							$uuidAttributes)
+					$uuidAttributes)
```
Comment (blizzz) [inline apps/user_ldap/lib/Connection.php:411]: the `in_array(…)` should be in one line, it looks strange spread over two

**Review by blizzz — APPROVED** [review]

**Comment by come-nc** [comment]
CI failure unrelated

**Review by artonge — APPROVED** [review]

**Pull request merged**